packages/compass-saved-aggregations-queries/src/stores/aggregations-queries-items.ts

```diff
@@ -18,21 +18,65 @@ export type Item = {
   name: string;
   database: string;
   collection: string;
-  type: 'query' | 'aggregation';
-};
+} & (
```

Comment:
All changes in this file are copied over from https://github.com/mongodb-js/compass/pull/2755